script/maven_overlay.sh

```diff
@@ -15,14 +15,94 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+location=$(dirname $0)
+rootdir=$(realpath ${location}/../)
+
+if [ "$#" -lt 2 ]; then
+  echo "usage: $0 <output directory> <Camel K runtime version> [<staging repository>] [<local Camel K runtime project directory>]"
```

Comment:
it's a small issue but the usage message doesn't match the actual code. Here `<output directory>` comes first but actually it should be `$2` arg.

Maybe I guess the usage message is the intended one and the code `output_dir=$2` is wrong?  Is it really meant to be :
```
output_dir=$1
runtime_version=$2
```
?

Reply:
yes, the usage message is wrong. many thanks!